tools/gitpod/workspace_config

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Should we move this under testing/ ?

Reply:
no - more details in https://github.com/jupyter/accessibility/pull/83#issuecomment-1123453188

Reply:
Sorry I meant the whole folder structure, as in:

    mv tools/ testing/.

Based on your file folder diagram below, I think we're on the same page ;)